python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -21,6 +20,14 @@
 from ray._private.gcs_utils import PlacementGroupTableData
 from ray.core.generated.common_pb2 import PlacementStrategy
 from ray.autoscaler._private.constants import AUTOSCALER_CONSERVE_GPU_NODES
+from ray.autoscaler._private.util import (
+    is_placement_group_resource,
```

Comment:
Fixing some of these backwards/circular dependency issues as they come up. (util.py importing resource_demand scheduler caused these changes)

Reply:
omgwhywouldutilimportresourcedemandscheduler